circle.yml

```diff
@@ -0,0 +1,12 @@
+general:
+  build_dir: ./
+checkout:
+  post:
+    - curl --location 'https://nuget.org/nuget.exe' -o ./.nuget/NuGet.exe
+    - xbuild /p:Configuration=Release ./Docker.DotNet.sln
```

Comment:
It will install mono before doing the Git pull, as I have it specified in the dependencies section (pre). Otherwise no, it wouldn't be available before the mono install.